sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/unix_time_helper.hpp

```diff
@@ -29,5 +29,17 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Common { n
     {
       return Azure::DateTime(1970) + std::chrono::seconds(unixTime);
     }
+
+    /**
+     * @brief Converts a #Azure::Core::Datetime to unix time.
+     *
+     * @param dateTime The date time to convert.
+     */
+    static inline uint64_t DatetimeToUnixTime(Azure::DateTime dateTime)
+    {
+      auto secondsSince1970
+          = std::chrono::duration_cast<std::chrono::seconds>(dateTime - Azure::DateTime(1970));
```

Comment:
nit: Can you add a comment explaining why we chose 1970?

```suggestion
      //  This count starts at the Unix Epoch which was January 1st, 1970 at UTC. 
      auto secondsSince1970
          = std::chrono::duration_cast<std::chrono::seconds>(dateTime - Azure::DateTime(1970));
```